modules/osdk-managing-psa-for-operators-with-escalated-permissions.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-managing-psa-for-operators-with-escalated-permissions.adoc_{context}"]
+= Managing pod security admission for Operator workloads that require escalated permissions
+
+If your Operator project requires escalated permissions to run, you cannot rely on pod security admission synchronization to label your Operator's namespaces. You must set your Operator's namespace label with the appropriate pod security level.
+
+[IMPORTANT]
+====
+If you do not set your Operator's namespace label, you must include namespace labelling as part of your Operator's manual installation instructions.
+====
+
+.Procedure
+
+. Set the security context constraint (SCC) to the required permission level as shown in the following example:
++
+.Example SCC with network administrator privileges
+[source,yaml]
+----
+containers:
+   - name: my-container
+     securityContext:
+       allowPrivilegeEscalation: false
+       capabilities:
+         add:
+           - "NET_ADMIN"
+----
+
+. Set the service account privileges to use the SCCs it needs to run your Operator's workloads as shown in the following example:
++
+[source,yaml]
+----
+ install:
```

Comment:
should make it clear these edits are done in the operator's clusterserviceversion yaml

Reply:
Addressed in dee0dc7